docs/user/lti.rst

```diff
@@ -84,7 +84,7 @@ With the Online Course setting enabled, you can now proceed to integrate Artemis
 
 1.2. Copy Dynamic Registration URL from Artemis
 """""""""""""""""""""""""""""""""""""""""""""""
-Once the above setting is enabled, you can now set up a bridge between Artemis and Moodle.
+Once the above setting is enabled, you can now request your Moodle Admin to set up a bridge between Artemis and Moodle. Moodle Admin needs course Dynamic Registration URL to successfully complete the configuration.
```

Comment:
_Moodle Admin needs **the** course Dynamic Registration_

Reply:
Thank you for noticing I added your suggestion 👍